cmd/walletd/miner.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"math/big"
+	"time"
+
+	"go.sia.tech/core/types"
+	"go.sia.tech/coreutils"
+	"go.sia.tech/walletd/api"
+	"lukechampine.com/frand"
+)
+
+func runCPUMiner(c *api.Client, minerAddr types.Address, n int) {
+	log.Println("Started mining into", minerAddr)
+	start := time.Now()
+
+	var last types.ChainIndex
+	var blocksFound int
+	for {
+		if n > 0 && blocksFound >= n {
+			break
+		}
+		elapsed := time.Since(start)
+		cs, err := c.ConsensusTipState()
+		check("Couldn't get consensus tip state:", err)
+		if cs.Index == last {
+			fmt.Println("Tip now", cs.Index)
+			last = cs.Index
+		}
```

Comment:
why set it if it already matches? Considering that `last` isn't used anywhere else this probably never prints?